service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/HmctsEmailContentProvider.java

```diff
@@ -19,9 +23,19 @@ public SubmitCaseHmctsTemplate buildHmctsSubmissionNotification(CaseData caseDat
         SubmitCaseHmctsTemplate template = buildNotifyTemplate(SubmitCaseHmctsTemplate.builder().build(), caseData);
 
         template.setCourt(courtService.getCourtName(caseData));
-        template.setLocalAuthority(laNameLookup.getLocalAuthorityName(caseData.getCaseLocalAuthority()));
+        template.setLocalAuthority(nonNull(caseData.getCaseLocalAuthority())
+            ? laNameLookup.getLocalAuthorityName(caseData.getCaseLocalAuthority()) : getApplicantName(caseData));
         template.setDocumentLink(getDocumentUrl(caseData.getC110A().getSubmittedForm()));
 
         return template;
     }
+
+    public String getApplicantName(CaseData caseData) {
+        LocalAuthority applicant = caseData.getLocalAuthorities().stream()
+            .map(Element::getValue)
+            .findFirst()
+            .orElse(null);
+
+        return nonNull(applicant) ? applicant.getName() : null;
+    }
```

Comment:
If the applicant is a solicitor can we not still display their name? Seems a lot of places that refer to the LA revert back to null in these cases, will that leave any blanks in the application/ cause any other effects?